SnakeAndLadder/src/DriverClass.java

```diff
@@ -0,0 +1,70 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class DriverClass {
+	public void playgame(Map<String,PlayerClass> playermap,Map<Integer,Integer> snakemap,Map<Integer,Integer> laddermap)
+	{
+		int flag=1;

```

Comment:
what is flag here?

Reply:
Flag to break the game if any player win the game

Reply:
use a better variable name reading which it should be clear what it actually means.